core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -1618,6 +1618,45 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "return 409 for copying entities into a workspace with subtree conflicts, but successfully copy when asked to" in withTestDataApiServices { services =>
```

Comment:
Please add asserts for the results returned in the existing copy-conflict tests (around lines 209 and 1609)